src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoderSettings.cs

```diff
@@ -28,6 +28,11 @@ public TextEncoderSettings()
         /// </summary>
         public TextEncoderSettings(TextEncoderSettings other)
         {
+            if (other == null)
+            {
+                throw new ArgumentNullException(nameof(other));
```

Comment:
Does it throw same exception on Desktop?

Reply:
@karelz  Desktop doesn't have this library.